src/test/regress/expected/multi_reference_table_2.out

```diff
@@ -0,0 +1,1606 @@
+SET citus.next_shard_id TO 1250000;
```

Comment:
Again, the diff is trivial and just removing the test with OIDS should be good enough:
```
< ERROR:  alter table command is currently unsupported
< DETAIL:  Only ADD|DROP COLUMN, SET|DROP NOT NULL, SET|DROP DEFAULT, ADD|DROP|VALIDATE CONSTRAINT, SET (), RESET (), ATTACH|DETACH PARTITION and TYPE subcommands are supported.
---
> ERROR:  syntax error at or near "WITH"
> LINE 1: ...ER TABLE reference_schema.reference_table_ddl SET WITH OIDS;
>        
```

Citus doesn't support WITH OIDS anyway, so I don't care the test much.